Dockerfile

```diff
@@ -52,4 +52,6 @@ RUN set -ex; \
         chmod +x /usr/bin/tini; \
     fi
 COPY --from=builder /opt/kroxylicious/kroxylicious-app/target/kroxylicious-app-${KROXYLICIOUS_VERSION}-bin/kroxylicious-app-${KROXYLICIOUS_VERSION}/ /opt/kroxylicious/
+COPY --from=builder /opt/kroxylicious/kroxylicious-additional-filters/kroxylicious-multitenant/target/kroxylicious-multitenant-${KROXYLICIOUS_VERSION}.jar /opt/kroxylicious/libs/
```

Comment:
is there some way to tie this to the module set used on line 12? Maybe using the assembly plugin? Doesn't need tackled in this PR as it could easily be a rabbits warren of issues.

Reply:
Sounds complicated, I don't know that we should invest too much in this idea of building it all with maven in one go. Since we will want the same mechanism for installing community filters and the Kroxylicious provided ones I imagine we will have to disconnect the two things.

Will open up a discussion.

Reply:
https://github.com/kroxylicious/design/discussions/36